scripts/notify_team_new_comment.js

```diff
@@ -0,0 +1,108 @@
+const fs = require('fs');
+const path = require('path');
+const { LE_BOT_USERNAME, BOT_MESSAGE } = require('./constants');
+const { CLOSE_CONTRIBUTORS } = require('./close_contributors');
+
+module.exports = async ({ github, context, core }) => {
+  try {
+    const issueNumber = context.payload.issue.number;
+    const issueUrl = context.payload.issue.html_url;
+    const issueTitle = context.payload.issue.title;
+    const escapedTitle = issueTitle.replace(/"/g, '\\"');
```

Comment:
This is not new, but the title is the only place I can think of where there might be user input that could be used in a way that might be used for an injection attack. I don't know there's much more we can do here though, and I think we have to assume that Slack have done their due dilligence with the webhook payloads.

Reply:
Thanks @rtibbles